src/main/java/RedisPool.java

```diff
@@ -0,0 +1,7 @@
+public class RedisPool extends ObjectPool {
```

Comment:
The class ObjectPool is typed, you don't pass it a type at this point, so it will have a wildcard (and might not compile in some stricter environments).

Reply:
done!